lib/livebook_web/controllers/auth_controller.ex

```diff
@@ -15,8 +15,14 @@ defmodule LivebookWeb.AuthController do
     end
   end
 
-  def index(conn, params) do
-    render(conn, "index.html", auth_mode: Livebook.Config.auth_mode(), errors: params["errors"])
+  def index(conn, %{"redirect_to" => path}) do
+    conn
+    |> put_session(:redirect_to, path)
+    |> redirect(to: current_path(conn, %{}))
+  end
+
+  def index(conn, _params) do
+    render(conn, "index.html", errors: [], auth_mode: Livebook.Config.auth_mode())
   end
```

Comment:
Do we still need these changes?

Reply:
Yup, we use `redirect_to` for the "login into Livebook" link, so we can redirect back to the app.

Reply:
perfect.